src/modules/Bootloader.rb

```diff
@@ -156,9 +156,20 @@ def Read
         ::Bootloader::BootloaderFactory.current.read
       rescue ::Bootloader::UnsupportedBootloader => e
         ret = Yast::Report.AnyQuestion(_("Unsupported Bootloader"),
-          _("Unsupported bootloader '%s' detected. Show proposal of supported configuration?") %
+          _("Unsupported bootloader '%s' detected. Use proposal of supported configuration instead?") %
             e.bootloader_name,
-          _("Show"),
+          _("Use"),
+          _("Quit"),
+          :yes) # focus proposing new one
+        return false unless ret
+
+        ::Bootloader::BootloaderFactory.current = ::Bootloader::BootloaderFactory.proposed
+        ::Bootloader::BootloaderFactory.current.propose
+      rescue ::Bootloader::BrokenConfiguration => e
```

Comment:
it depends if configuration information is needed during read or later when trying to understand it ( like e.g. udev can be broken when read in various places, but if it is in stage1 location, then it is read as value, but translated from udev later when we try to interpret if it is e.g. MBR device or not ).